tests/robot-tests/tests/libs/utilities.py

```diff
@@ -163,31 +163,40 @@ def parent_does_not_contain_matching_elements() -> bool:
         raise_assertion_error(err)
 
 
-def get_child_element(parent_locator: object, child_locator: str):
-    try:
-        children = get_child_elements(parent_locator, child_locator)
-
-        if len(children) == 0:
-            raise_assertion_error(
-                f"Found no elements matching child locator {child_locator} under parent "
-                f"locator {parent_locator} in utilities.py#get_child_element()"
-            )
-
-        if len(children) > 1:
-            logger.warn(
-                f"Found {len(children)} child elements matching child locator {child_locator} "
-                f"under parent locator {parent_locator} in utilities.py#get_child_element() - "
-                f"was expecting only one. Consider making the parent selector more specific. "
-                f"Returning the first element found."
-            )
+def get_child_element(parent_locator: object, child_locator: str, retries: int = 5, delay: float = 1.0):
+    def log_and_raise(message: str):
+        logger.warning(message)
+        raise AssertionError(message)
```

Comment:
Rather than `raise AssertionError(message)`, I think this should be `raise_assertion_error(message)`.

Reply:
I have completed removed above lines and made use of existing raise_assertion_error method defined from line 84-86